tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -150,62 +159,99 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+            var tooltipContent = [];
+            var pageStartIndex = i;
+            var pageEndIndex = (i + pageSize -1) < listOfData.length ? (i + pageSize -1) : (listOfData.length - 1);
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageStartIndex].text});
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageEndIndex].text});
+            tooltipContents.push(tooltipContent);
+        }
+
+        return tooltipContents;
+    };
+
+    fluid.tests.verifyLinksAndTooltips = function(pager, pageSize, linksList) {
+        fluid.tests.setPageSize(pager, pageSize);
+        fluid.tests.verifyPagerBar(pager, pageSize, "top");
+        fluid.tests.verifyPagerBar(pager, pageSize, "bottom");
+        fluid.tests.verifyToolTipsOfPagerBar(pager, pageSize, "top", fluid.tests.getTooltipContents(linksList, pageSize));
+        fluid.tests.verifyToolTipsOfPagerBar(pager, 4, "bottom", fluid.tests.getTooltipContents(linksList, pageSize));
+    };
+
+    fluid.tests.assertVisibleTips = function (pager, message, targetIds, expectedTooltip) {
+        fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
+            jqUnit.assertNode(tooltip.length, expectedTooltip, $("b", tooltip));
+        });
+    };
+
+    fluid.tests.setPageSize = function (pager, pageSize) {
+        fluid.changeElementValue(pager.locate("pageSize"), pageSize);
+    };
+
+    fluid.tests.verifyPagerBar = function (pager, pageSize, location) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
+
+        //Verify the number of links in the pager bar
+        jqUnit.assertEquals("The number of links in the pager should be according to the page size",
+            expectedNumberOfLinks,
+            pager[location === "top" ? "pagerBar" : "pagerBar-1"].locate("pageLinks").length);
+    };
+
+    fluid.tests.verifyToolTipsOfPagerBar = function (pager, pageSize, location, expectedTooltips) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
+
+        for (var x = 0; x < expectedNumberOfLinks; x++) {
+            fluid.tests.verifyPagerToolTip(pager, location, x, expectedTooltips[x]);
+        }
+    };
+
+    fluid.tests.verifyPagerToolTip = function (pager, location, index, expectedTooltip) {
+        var link =  $(pager[location === "top" ? "pagerBar" : "pagerBar-1"].locate("pageLinks").find("a")[index]);
+        var linkNumber = index + 1;
+        var linkId = "page-link:link" + linkNumber + (location === "bottom" ? "-1" : "");
+
+        link.trigger("focus");
+        fluid.tests.assertVisibleTips(pager,
+            "The tooltip of page link " + linkNumber + ", in " + location + " page bar is visible",
+            [linkId], expectedTooltip);
+
+        link.trigger("blur");
+        fluid.tests.assertVisibleTips(pager,
+            "There shouldn't have any tooltips visible when none of the pageLinks are focused",
```

Comment:
Reword this as "There should not be any tooltips visible when no pageLinks are focused."